API/DTOs/OrderDTO.cs

```diff
@@ -7,8 +8,11 @@ public class OrderDTO
     {
         public Guid? Id { get; set; }
         [Required]
-        public OrderTypeDTO? OrderType { get; set; }
-        public ProductOrderDTO? ProductOrder { get; set; }
-        public ServiceOrderDTO? ServiceOrder { get; set; }
+        public Guid? CustomerId { get; set; }
+        [Required]
+        public IEnumerable<OrderStatusEnum>? Status { get; set; }
+        public IEnumerable<CurrencyEnum>? Currency { get; set; }
```

Comment:
Aš šiek tiek nesupranti, kodėl čia Currency yra kaip IEnumerable. Ar orderio galutinė suma gali turėti kelias skirtingas valiutas?

Reply:
Aš čia pagal jų tą .yaml šitą įdėjau, bet jo, nelogiška gal.

Reply:
Gali ištrint tada, nes tikrai nesąmonė

Reply:
O kaip tada siūlai juos apsirašyt?

Reply:
```
public CurrencyEnum? Currency { get; set; }
```